src/components/story-block.js

```diff
@@ -14,14 +14,16 @@ const StoryBlock = ({
   imageRight = false,
 }) => (
   <section>
-    <Container>
+    <Container className="my-1">
       <Row>
         <Col sm={{ span: '6', order: `${imageRight ? 'last' : 'first'} ` }}>
           <Img fluid={imageSource} alt={altText} />
         </Col>
         <Col sm={6}>
-          {description}
-          <ReactMarkdown source={fullText} />
+          <p className="font-weight-bold">{description}</p>
```

Comment:
@ToniTepes im setting this as bold here, so actually it should've need to be in your CSS at all once we merge